examples/01-basic/04-default-blocks/App.tsx

```diff
@@ -32,6 +32,11 @@ export default function App() {
         type: "heading",
         content: "Heading",
       },
+      {
+        type: "heading",
+        props: { isTogglable: true },
```

Comment:
let's use isToggleable to be more inline with notion (https://developers.notion.com/reference/block#toggle-blocks)